lib/shopify-cli/forms/create_app.rb

```diff
@@ -17,7 +17,7 @@ def ask
       private
 
       def fallback_title
-        name.gsub(/(.)([A-Z])/, '\1 \2') # change camelcase to title
+        name.gsub(/(.)([A-Z])/, '\1\2') # change camelcase to title
```

Comment:
This is mean to check for a capital letter with something proceeding it then add a space in-between. You have taken out the space so essentially this does nothing now. The tests still passed after this?